ohsome/clients.py

```diff
@@ -59,6 +63,15 @@ def __init__(
             agent_list.append(user_agent)
         self.user_agent = " ".join(agent_list)
 
+        if retry is None:
+            self.__retry = Retry(
+                total=3,
+                status_forcelist=[429, 500, 502, 503, 504],
+                allowed_methods=["GET", "POST"],
+                backoff_factor=1,
+            )
+        else:
+            self.__retry = retry
```

Comment:
```suggestion
            self.__retry = retry or Retry(
                total=3,
                status_forcelist=[429, 500, 502, 503, 504],
                allowed_methods=["GET", "POST"],
                backoff_factor=1,
            )
```

Reply:
this is what I had until but then wanted to allow `retry=False`, which is valid. But lets stick to this one, its nicer.